### PR TITLE
Limit code repetition in `app/views.py`

### PR DESCRIPTION
**Pull request by HirumalPriyashan**
As [this issue](https://github.com/spdx/spdx-online-tools/issues/184) points out, there are redundant code segments in the `app/views.py`. This PR will removes few redundant code segments from there.
Additionally, have removed unused imports from `app/views.py`.

### CODE REVIEW

File: src/app/views.py [hunk 1]
```diff
@@ -857,58 +851,46 @@ def license_xml_edit(request, page_id):
     else:
         return HttpResponseRedirect('/app/xml_upload')
 
+
+def get_context_dict(request, license_obj):
```
Comment (rtgdk) [inline src/app/views.py:855]: Let's rename it to get_context_dict_for_license_xml

File: src/app/views.py [hunk 2]
```diff
@@ -857,58 +851,46 @@ def license_xml_edit(request, page_id):
...
-        context_dict["xml_text"] = license_obj.xml
-        context_dict["license_name"] = license_obj.fullname
-        return render(request,
-            'app/editor.html',context_dict,status=200
-            )
+        context_dict = get_context_dict(request, license_obj)
+        return render(request, "app/editor.html", context_dict, status=200)
     else:
-        return HttpResponseRedirect('/app/license_requests')
+        return HttpResponseRedirect("/app/license_requests")
```
Comment (rtgdk) [inline src/app/views.py:879]: why this change is required?

File: src/app/views.py [hunk 3]
```diff
@@ -1205,112 +1187,84 @@ def issue(request):
...
 
 
+def pull_request(request):
+    """ View that handles pull request """
+    return handle_pull_request(request, is_ns=False)
+
+
 def namespace_pull_request(request):
     """ View that handles pull request for a license namespace """
-    if request.user.is_authenticated:
```
Comment (rtgdk) [inline src/app/views.py:1264]: Good catch for merging these!

File: src/app/views.py (hunk 2 above)
Comment (HirumalPriyashan) [reply src/app/views.py:879]: It is due to the formatting I have done to the code. If single quotes are more preferred than double quotes I can revert that change.

Comment (rtgdk) [reply src/app/views.py:879]: Let's keep the same formatting for now and minimal changes.

There's a different issue tracking for adding linters to the repository which should remove these issues.

Comment (HirumalPriyashan) [reply src/app/views.py]: Updated with [d53ed24](https://github.com/spdx/spdx-online-tools/pull/441/commits/d53ed2496ae73af90a6e26f6ccef74378e2511de).

**Comment by rtgdk** [comment]
Overall LGTM @HirumalPriyashan . Thanks for working on this!

Let's run tests on these change and post the screenshot. Will merge after that.
`python manage.py test`

**Review by rtgdk — APPROVED** [review]

**Comment by HirumalPriyashan** [comment]
Running all the tests through `python manage.py test` already producing errors. Which I have mentioned [here](https://github.com/spdx/spdx-online-tools/pull/438#issuecomment-1463442628).

Have check which tests are depend on the changes and checked whether they are passing.
image

**Comment by rtgdk** [comment]
Thanks @HirumalPriyashan for the contribution
